diag/exec/controller/exec_controller.cpp

```diff
@@ -46,6 +46,10 @@ ExecController::ExecController() {}
 ExecController::~ExecController() {
 }
 
+void ExecController::ForceStop() {
+    this->thread_.request_stop();
+}
+
```

Comment:
po co to jest ?.

Reply:
żeby zatrzymać wysyłanie hb przez aplikacje